lms/services/lti_outcomes.py

```diff
@@ -1,26 +1,83 @@
+from datetime import datetime, timezone
 from xml.parsers.expat import ExpatError
 
 import xmltodict
 
+from lms.services import LTIAHTTPService
 from lms.services.exceptions import ExternalRequestError
 
 __all__ = ["LTIOutcomesClient"]
 
+SCOPES = [
+    "https://purl.imsglobal.org/spec/lti-ags/scope/lineitem",
+    "https://purl.imsglobal.org/spec/lti-ags/scope/result.readonly",
+    "https://purl.imsglobal.org/spec/lti-ags/scope/score",
+]
+
 
 class LTIOutcomesClient:
     """
     Service for making requests to an LMS's Outcomes Management endpoint.
 
-    See https://www.imsglobal.org/specs/ltiomv1p0/specification.
+    See:
+         LTI1.1 https://www.imsglobal.org/specs/ltiomv1p0/specification.
     """
 
-    def __init__(self, _context, request):
-        self.oauth1_service = request.find_service(name="oauth1")
-        self.http_service = request.find_service(name="http")
+    def __init__(  # pylint:disable=too-many-arguments
```

Comment:
Changed this service to get the services injected and use a factory.

It's just how we tend to to thing now and allows to create a the service from scratch if necessary (for example in `make shell` which was necessary for the certification testing)